autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -194,6 +192,7 @@ def get_max_level_all(self):
             max_level = max(max_level, self.get_max_level(stack_name))
         return max_level
 
+    # TODO: Rename method for v0.1
     def get_models(self, hyperparameters, hyperparameter_tune=False, **kwargs):
```

Comment:
add documentation since this method's purpose could be easily misinterpreted?

Reply:
added